kernelci/config/rootfs.py

```diff
@@ -94,6 +95,9 @@ class RootFSFactory(YAMLObject):
     @classmethod
     def from_yaml(cls, name, rootfs):
         rootfs_type = rootfs.get('rootfs_type')
+        if rootfs_type not in cls._rootfs_types:
+            print("Unsupported rootfs_type {}".format(rootfs_type))
+            sys.exit(1)
```

Comment:
If you want to abort the program then throw an exception instead of `print` and `sys.exit(1)`.

Also you can check `if rootfs_type is None` since that would lead to a different type of error (i.e. no rootfs type specificed).